stacks/patterns/home.php

```diff
@@ -3,109 +3,70 @@
  * Title: Stacks
  * Slug: stacks/home
  * Block Types: core/post-content
- * Post Types: post, page
```

Comment:
Why did you remove this? The idea is that this will prompt you to add the pattern when you create a new post or page.

Reply:
Related to the point about single page sites, this was stopping the pattern from being loaded in home.html so the template was blank.

Reply:
Still shows in post editor doesn't it? I thought that attribute just dictates where that pattern is available, and by defining post/page you're stating it shouldn't be accessible in site editor. 

<img width="972" alt="image" src="https://user-images.githubusercontent.com/1072756/228102091-ebfff0fe-0794-42e4-9214-4b97c263fa44.png">

Reply:
Hmm, I don't see it...

Reply:
Btw, if we don't chose the pattern and write regular content for a page, the frontend just shows a white screen